DuckDuckGo/CrashCollectionOnboardingView.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  CrashCollectionOnboardingView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import DuckUI
+import DesignResourcesKit
+
+struct CrashCollectionOnboardingView: View {
+
+    @ObservedObject var model: CrashCollectionOnboardingViewModel
+
+    var body: some View {
+        if #available(iOS 16.0, *) {
+            // Using NavigationStack here in order to fix presentation on the iPad, where `Navigation` would use Split Navigation view.
+            NavigationStack {
+                contents
+            }
+            .background(Color(designSystemColor: .backgroundSheets))
+        } else {
+            NavigationView {
+                contents
+            }
+            .background(Color(designSystemColor: .backgroundSheets))
+        }
+    }
+
+    var contents: some View {
+        VStack(spacing: 0) {
+            ScrollView {
+                VStack(spacing: 24) {
+                    Image("Breakage-128")
```

Comment:
We could use [`Image(decorative:)`](https://developer.apple.com/documentation/swiftui/image/init(decorative:bundle:)) here to make sure that VoiceOver API doesn't read the image name (it does this in the current implementation).

Reply:
Of course :) thanks for pointing that out :) I'll fix it right away.